lib/cup.rb

```diff
@@ -0,0 +1,22 @@
+
+module Cup 
```

Comment:
It's nice that you are trying something new, but there's really no use for a module here. I would prefer you use the more fundamental skills we've been covering throughout the mod